src/js/common/components/RequiredLoginView.jsx

```diff
@@ -103,7 +103,7 @@ RequiredLoginView.propTypes = {
   authRequired: PropTypes.number.isRequired,
   serviceRequired: PropTypes.string.isRequired,
   userProfile: PropTypes.object.isRequired,
-  loginUrl: PropTypes.string,
+  loginUrl: PropTypes.object.isRequired
```

Comment:
I'd like for  `LoginPrompt`, `VerifyPrompt` and `AuthApplicationSection` require their url props (either `loginUrl` or `verifyUrl`, but because of the async fetch, they are not always available by the time the component is created. So, I've removed all of the requirements and left the check in the handlers to make sure these are well-defined before popping open the ID.me window.